apps/web/src/app/[orgShortCode]/convo/_components/create-convo-form.tsx

```diff
@@ -290,7 +292,7 @@
     return false;
   }, [isTextPresent, topic, selectedParticipants, selectedEmailIdentity]);
 
-  async function startConvoUnderlying(type: 'message' | 'comment') {
+  async function startConvoCreation(type: 'message' | 'comment') {
     const getPublicIdsByType = (
       type: NewConvoParticipant['type'],
       property = 'publicId'
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 301]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->@BlankParticle there is a type error on this line, can you check and fix please<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/un/inbox/518?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->